packages/x-charts-pro/src/context/ZoomProvider/useSetupPan.ts

```diff
@@ -106,7 +106,7 @@ export const useSetupPan = () => {
 
     const handleUp = (event: PointerEvent) => {
       eventCacheRef.current.splice(
-        eventCacheRef.current.findIndex((e) => e.pointerId === event.pointerId),
+        eventCacheRef.current.findIndex((cachedIndex) => cachedIndex.pointerId === event.pointerId),
```

Comment:
Assuming you want the same as below?

```suggestion
        eventCacheRef.current.findIndex((cachedEvent) => cachedEvent.pointerId === event.pointerId),
```